include/NovelRT/NovelRunner.h

```diff
@@ -14,7 +14,13 @@ namespace NovelRT {
   class NovelRunner {
 
   public:
+    /**
+     * An event that occurs when the scene has to be (re)constructed.
```

Comment:
I think better wording might be `This event is used for constructing the rendered scene. This is called once per frame update.`

Reply:
Will change